### PR TITLE
Update jenkinsfileBase to enchance check condition for CUTOMIZED_SDK_URL

### PR DESCRIPTION
**Pull request by YeeSkywalker**
Checkout: Yee Lin
Date: Jan 21st, 2022
Issue: https://github.com/adoptium/aqa-tests/issues/2097

### CODE REVIEW

**Comment by YeeSkywalker** [comment]
grind output: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3173/console

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -368,12 +368,12 @@ def setup() {
 			setupEnv()
 
 			if (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL) {
-				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
-			} else if (params.CUSTOMIZED_SDK_URL) {
-				SDK_RESOURCE = "customized"
+			}else if (params.SDK_RESOURCE != 'customized'){
```
Comment (sophia-guo) [inline buildenv/jenkins/JenkinsfileBase:371]: `CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"` is missing in the if block (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL)

**Comment by sophia-guo** [comment]
In your grinder parameter is set as `www.google.com`?

**Comment by llxia** [comment]
Error in the Grinder:
```
[Pipeline] echo
Exception: groovy.lang.MissingPropertyException: No such property: CUSTOMIZED_SDK_URL_OPTION for class: Script1
[Pipeline] }
```

Please see [Sophia's comment](https://github.com/adoptium/aqa-tests/pull/3260#discussion_r790057559) above.

**Pull request closed without merging**